### PR TITLE
zh-TW: Translate docs index

### PR DESCRIPTION
**Pull request by osk2**
Translate `zh-tw/docs/index.md`

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/nodejs-tw, please have a review, thanks!

File: locale/zh-tw/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* [Node.js 0.10.x](https://nodejs.org/docs/latest-v0.10.x/api/)
+* [所有版本](https://nodejs.org/docs/)
+
+</div>
+
+### ES6 功能
+
+[ES6 章節](/zh-tw/docs/es6/)中介紹了三個 ES6 的特色群組以及 Node.js 預設啟用哪些功能的細節並附上連結。此章節也示範了如何查詢某版 Node.js 中的 V8 版本。
+
+### 教學
```
Comment (PeterDaveHello) [inline locale/zh-tw/docs/index.md]: Wrong hierachy

File: locale/zh-tw/docs/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* [Node.js 6.x](https://nodejs.org/docs/latest-v6.x/api/)
+* [Node.js 5.x](https://nodejs.org/docs/latest-v5.x/api/)
+* [Node.js 4.x](https://nodejs.org/docs/latest-v4.x/api/)
+* [Node.js 0.12.x](https://nodejs.org/docs/latest-v0.12.x/api/)
+* [Node.js 0.10.x](https://nodejs.org/docs/latest-v0.10.x/api/)
+* [所有版本](https://nodejs.org/docs/)
+
+</div>
+
+### ES6 功能
```
Comment (PeterDaveHello) [inline locale/zh-tw/docs/index.md]: Wrong hierachy

File: locale/zh-tw/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## API 參考文件
+
+[API 參考文件](https://nodejs.org/api/)提供了 Node.js 中的函式或物件的詳細資訊。此文件詳述了一個方法可接受哪些參數、其回傳值及哪些錯誤可能與之關聯，同時也記錄了在哪些 Node.js 版本中可使用此方法。
+
+此文件僅會介紹由 Node.js 官方內建的模組，社群提供的模組則不在此列。
+
+<div class="highlight-box">
+
+### 在找舊版本的 API 文件嗎？
```
Comment (PeterDaveHello) [inline locale/zh-tw/docs/index.md]: Suggestion:
```diff
---  ### 在找舊版本的 API 文件嗎？
+++  ### 在找舊版的 API 文件嗎？
```

File: locale/zh-tw/docs/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  lts: LTS
+---
+
+# 關於文件
+
+本站提供下列幾種類型的文件：
+
+* API 參考文件
+* ES6 功能
+* 教學
```
Comment (PeterDaveHello) [inline locale/zh-tw/docs/index.md]: This seems to be another word?

**Review by PeterDaveHello — CHANGES_REQUESTED** [review]

File: locale/zh-tw/docs/index.md (hunk 4 above)
Comment (osk2) [reply locale/zh-tw/docs/index.md]: How about "技術指南"?

**Review by willin — APPROVED** [review]
LGTM

File: locale/zh-tw/docs/index.md (hunk 4 above)
Comment (PeterDaveHello) [reply locale/zh-tw/docs/index.md]: Looks great!

**Review by PeterDaveHello — APPROVED** [review]

**Pull request merged**